src/mower_utils/test/README.md

```
@@ -0,0 +1,68 @@
+cd ~/catkin_ws
+catkin_make tests
+source devel/setup.bash
+rostest mower_utils coverage_paths.test            # results in build/test_results/
+rostest mower_utils coverage_paths.test --text     # also stream logs to stdout
+```
+
+Wall time ≈ 4 min for the 9-area sample fixture. Failures are listed per
+area with the first bad pose, the worst cell value (0 = free, 100 = lethal),
+and the path index within that area:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)